src/source/Crypto/Tls_mbedtls.c

```diff
@@ -26,9 +28,17 @@ STATUS createTlsSession(PTlsSessionCallbacks pCallbacks, PTlsSession* ppTlsSessi
     mbedtls_ssl_config_init(&pTlsSession->sslCtxConfig);
     mbedtls_ssl_init(&pTlsSession->sslCtx);
     CHK(mbedtls_ctr_drbg_seed(&pTlsSession->ctrDrbg, mbedtls_entropy_func, &pTlsSession->entropy, NULL, 0) == 0, STATUS_CREATE_SSL_FAILED);
-    CHK(mbedtls_x509_crt_parse_file(&pTlsSession->cacert, KVS_CA_CERT_PATH) == 0, STATUS_INVALID_CA_CERT_PATH);
+
+    // Read and parse CA certificate
```

Comment:
could this be moved to a separate fn? e.g. ` CHK_STATUS(readAndParseCACertificate(pTlsSession));`

Reply:
Good suggestion, will create a static function which handles this.